index.js

```diff
@@ -57,13 +57,17 @@ const processPinedMessage = (message) => {
 // /poll
 bot.onText(/^\/poll(?:@.*?)?( .*|$)/, function(msg, match) {
   const chatId = msg.chat.id;
-  const username = msg.from.username;
-  let time = '';
-  if (msg.text.split(' ')[1]) {
-    time = msg.text.split(' ')[1];
-  }
+
+  // @<username> if users have set up their username;
+  // <first_name> if users have not set up their username.
+  const userIdentifier = msg.from.username
+    ? `@${msg.from.username}`
+    : msg.from.first_name;
```

Comment:
I think ${msg.from.first_name} ${msg.from.last_name} is better then only msg.from.first_name